### ISSUE TITLE
es6 not working 

### ISSUE THREAD

**Issue by chiragkanhasoft** [satendra02/react-chrome-extension#14]
i create a function like this in content.js file

` a =()=>{
      console.log("asd")
    }`

when i do build it gives me error like this.

```
Missing class properties transform.
  31 | class Main extends React.Component {
  32 |   
> 33 |     a =()=>{
     |     ^
  34 |       console.log("asd")
  35 |     }
  36 | 
 @ multi ./config/polyfills.js ./src/content.js content[1]

```

**Comment by satendra02** [comment]
Hi @chiragkanhasoft,
Can you post whole `content.js` file here? I will look at it.

**Comment by kevinchou723** [comment]
What was the solution to this issue?

I have node 14 installed

```
"dependencies": {
    "@babel/runtime": "^7.10.4",
    "autoprefixer": "^9.8.5",
    "case-sensitive-paths-webpack-plugin": "^2.3.0",
    "chalk": "^4.1.0",
    "css-loader": "^3.6.0",
    "dotenv": "^8.2.0",
    "dotenv-expand": "^5.1.0",
    "fs-extra": "^9.0.1",
    "postcss-flexbugs-fixes": "^4.2.1",
    "promise": "8.0.1",
    "prop-types": "^15.7.2",
    "raf": "3.4.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-frame-component": "^4.1.2",
    "resolve": "^1.17.0",
    "styled-components": "^5.2.1",
    "whatwg-fetch": "3.2.0"
  },
  "devDependencies": {
    "@babel/core": "^7.10.4",
    "@babel/plugin-proposal-class-properties": "^7.12.1",
    "@babel/plugin-transform-runtime": "^7.12.1",
    "@babel/preset-env": "^7.10.4",
    "@babel/preset-react": "^7.10.4",
    "@typescript-eslint/eslint-plugin": "^3.6.0",
    "@typescript-eslint/parser": "^3.6.0",
    "babel-eslint": "^10.1.0",
    "babel-jest": "^26.1.0",
    "babel-loader": "^8.1.0",
    "babel-preset-react-app": "^9.1.2",
    "eslint": "^7.4.0",
    "eslint-config-react-app": "^5.2.1",
    "eslint-loader": "^4.0.2",
    "eslint-plugin-flowtype": "^5.2.0",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-react": "^7.20.3",
    "eslint-plugin-react-hooks": "^4.0.7",
    "file-loader": "^6.0.0",
    "html-webpack-plugin": "^4.3.0",
    "jest": "^26.1.0",
    "mini-css-extract-plugin": "^0.9.0",
    "node-sass": "^4.14.1",
    "npm-watch": "^0.6.0",
    "postcss-loader": "^3.0.0",
    "react-dev-utils": "^10.2.1",
    "sass-loader": "^10.0.3",
    "style-loader": "^1.2.1",
    "sw-precache-webpack-plugin": "^1.0.0",
    "url-loader": "^4.1.0",
    "webpack": "^4.43.0",
    "webpack-cli": "^3.3.12",
    "webpack-dev-server": "^3.11.0",
    "webpack-manifest-plugin": "^2.2.0"
  }
````
